packages/cyberstorm-forms/src/components/FormSelect.module.css

```diff
@@ -0,0 +1,64 @@
+.SelectTrigger {
```

Comment:
This should use camelCase, not PascalCase, to be consistent with the Cyberstorm package. Applies to other selectors in this files as well.

EDIT: to think of it, these are all used in Select component, so I need no reason to use the `Select` prefix in the names at all.